samples/dotnet-hook-samples/Test.csproj

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
```

Comment:
Mismatch in spaces/tabs indentation.

Reply:
ah darn I thought I got rid of that already!